### PR TITLE
Added abstract and updated description

### PR DESCRIPTION
**Pull request by jasondunsmore**

### CODE REVIEW

**Comment by raj-patel** [comment]
jason, we dont need abstract in the template description.

File: wordpress-multi-server.yaml [hunk 1]
```diff
@@ -1,8 +1,15 @@
...
-  This is a Heat template to deploy Load Balanced WordPress servers with a
-  backend database server.
+  Multiple [WordPress](http://wordpress.org/about/) v3.9.2 servers
+  with [lsyncd](https://code.google.com/p/lsyncd/),
+  [vsftpd](https://security.appspot.com/vsftpd.html#about) and
+  [Varnish](https://www.varnish-cache.org/about) behind a Cloud Load
+  Balancer. The database will be hosted on a Linux server with [MySQL
+  5.5](http://www.mysql.com/about/).
+
+abstract: |
```
Comment (adampickeral) [inline wordpress-multi-server.yaml]: this line stays in the rackspace.yaml file

File: wordpress-multi-server.yaml [hunk 2]
```diff
@@ -1,8 +1,12 @@
 heat_template_version: 2013-05-23
 
 description: |
-  This is a Heat template to deploy Load Balanced WordPress servers with a
-  backend database server.
+  Multiple [WordPress](https://wordpress.org/about) v3.9.2 servers with
```
Comment (adampickeral) [inline wordpress-multi-server.yaml]: We'd also like for all of these descriptions to have heading's too:
```
#### Multiple Servers
```
which when parsed looks like:
#### Multiple Servers

**Comment by adampickeral** [comment]
:+1:

**Comment by jasondunsmore** [comment]
CircleCI is failing due to OverLimit error.

**Pull request merged**